README.md

```diff
@@ -11,13 +11,22 @@
 
 # Webpack CLI
 
-This project intends to be the main cli package of webpack. Here we will encapsulate all the features and code related to the command line. From sending options to the compiler, to initialize and migrate from version to version.
+This project intends to be the main cli package of webpack. Here we will encapsulate all the features and code related to the command line. From sending options to the compiler, to initialize and migrate from version to version. To avoid breaking changes and making people unhappy, we are keeping the old features. While we keep the old features, we've added some other, `init`and `migrate`. 
 
 **Note** The package is still in work in progress. In case you want to contribute, reach to us, so we can point you out how and when you can help us.
 
+## Migration from webpack v1 to v2
 
-# Roadmap to the first release
+The `migrate` feature aims to ease the transition version 1, to version 2 of webpack. This feature will allow users to switch to the new version of webpack, without having to refactor themselves.
```

Comment:
Better just to go with "eases" over "aims to ease". Same with "will". Better use "allows".